source/assets/components/RecipeUpload.js

```diff
@@ -78,6 +92,7 @@ class RecipeUpload extends HTMLElement {
         <div id="formButtons">
             <input id="submitButton" class="Submit" type="button" value="Submit">
             <input id="cancelButton" class="Cancel" type="button" value="Cancel">
+            <input id="deleteButton" class="Delete" type="button" value="Delete">
```

Comment:
The reason why i didn't add the delete button to the html immediately is bc it will show up in the create recipe page which wouldn't make sense.